Files/Scripts/Zelda 64.psm1

```diff
@@ -1272,8 +1272,8 @@ function CreateSwordTrailColorOptions() {
 
     CreateReduxGroup -Tag  "Colors" -Text "Sword Trail Colors"
                      CreateReduxComboBox -Name "SwordTrail"         -Text "Sword Trail Color"    -Items @("White", "Red", "Green", "Blue", "Cyan", "Magenta", "Orange", "Gold", "Purple", "Pink", "Randomized", "Custom") -Default 1 -Info ("Select a preset for the sword trail color" + $randomize) -Credits "Ported from Rando"
-    $buttons +=      CreateReduxButton   -Tag $Buttons.Count        -Text "Trail (Inner)"                                                                                                                                            -Info "Select the inner color you want for the sword trail"      -Credits "Ported from Rando"
```

Comment:
Do not change this, since I don't like the tip alpha option

Reply:
This is actually referring to the correction of the sword trail **color** and not the transparency of the trail here actually. I will show this too.